python/ml4ir/base/model/relevance_model.py

```diff
@@ -605,6 +606,9 @@ def evaluate(
             Value representing how often(in batches) to log status
         compute_intermediate_stats : bool
             Determines if group metrics and other intermediate stats on the test set should be computed
+        monte_carlo_inference_trials: int
```

Comment:
@mohazahran mentioned somewhere else to move to model config to keep this clean.